spatial/r3/vector.go

```diff
@@ -79,3 +92,46 @@ func Cos(p, q Vec) float64 {
 type Box struct {
 	Min, Max Vec
 }
+
+// TODO:
+//  - create rotations from Euler angles (NewRotationFromEuler?)
+//  - create rotations from rotation matrices (NewRotationFromMatrix?)
+//  - return the equivalent Euler angles from a Rotation
+//  - return the equivalent rotation matrix from a Rotation
```

Comment:
I am ambivalent about including Euler angles from quaternions. I would be OK with adding an example for how to do it, but it's really a bad approach to handling spatial rotation and I don't really want to encourage it.

Reply:
I'll reword so as to not sound as a "hard" TODO but merely as a set of possible suggestions.